src/app/admin/_components/specialist/SpecialistCreate/SpecialistCreateDetails.js

```diff
@@ -0,0 +1,50 @@
+'use client';
+
+import React from 'react';
+import { NumberInput, required, SelectInput } from 'react-admin';
+import { FormatOfWork, Gender } from '@/app/admin/_lib/consts';
+import { getChoicesList } from '@/app/admin/_utils/getChoicesList';
+import { FormFieldWrapper } from '@/app/admin/_components/shared/FormFieldWrapper';
+import {
+  SpecialistCreateFormBlocks,
+  SpecialistFormFields,
+} from '@/app/admin/_lib/specialistData';
+
+const SpecialistCreateDetails = () => {
+  const genderChoicesList = getChoicesList(Object.values(Gender));
+  const formatOfWorkChoicesList = getChoicesList(Object.values(FormatOfWork));
+
+  return (
+    <FormFieldWrapper
+      title={SpecialistCreateFormBlocks.details}
+      className="mt-5"
+    >
+      <div className="flex w-full gap-6">
+        <SelectInput
+          name={SpecialistFormFields.gender.name}
+          source={SpecialistFormFields.gender.name}
+          label={SpecialistFormFields.gender.label}
+          choices={genderChoicesList}
+          validate={required()}
```

Comment:
Dont we have zod validations?

Reply:
This attribute is required to put "*" next to the label. It doesn't affects zod validation which is going to be the main one.


```js
export const UserCreate = () => (
    <Create>
        <SimpleForm validate={validateUserCreation}>
            {/* 

                We need to add `validate={required()}` on required fields to append a '*' symbol 
                to the label, but the real validation still happens in `validateUserCreation`

            */}
            <TextInput label="First Name" source="firstName" validate={required()} />
            <TextInput label="Age" source="age" validate={required()} />
            <ArrayInput label="Children" source="children" fullWidth validate={required()}>
                <SimpleFormIterator>
                    <TextInput label="First Name" source="firstName" validate={required()} />
                    <TextInput label="Age" source="age" validate={required()} />
                </SimpleFormIterator>
            </ArrayInput>
        </SimpleForm>
    </Create>
);
```